common/common-utils/src/main/java/com/duckduckgo/common/utils/extensions/FragmentExtensions.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.common.utils.extensions
+
+import android.widget.EditText
+import androidx.fragment.app.Fragment
+
+fun Fragment.showKeyboard(editText: EditText) = requireActivity().showKeyboard(editText)
+
+fun Fragment.hideKeyboard(editText: EditText) = requireActivity().showKeyboard(editText)
```

Comment:
This should call `hideKeyboard`, not `showKeyboard`

Reply:
Fixed! Thanks 🙏